### ISSUE TITLE
Display selected applicant's OIDC claims in UI for handler

### ISSUE THREAD

**Issue by jaakkocsc** [CSCfi/rems#2241]
Handler UI should display selected OIDC claims returned by the OP. Selection could be static in config or configurable in admin UI.

**Comment by opqdonut** [comment]
duplicate of #2130 ?

**Comment by opqdonut** [comment]
possible after #2130
